java/org/apache/tomcat/util/scan/JarIndex.java

```diff
@@ -0,0 +1,61 @@
+package org.apache.tomcat.util.scan;
```

Comment:
No ALv2 Header
Why is this class in this package when it is only used in `org.apache.catalina.webresources`?